src/main/java/gov/usgs/wma/waterdata/groundwater/RdbWriter.java

```diff
@@ -38,10 +38,10 @@ public RdbWriter(Writer destination) {
 		initRows();
 	}
 
-	public long getHeaderRows() {
+	public long getHeaderRowCount() {
```

Comment:
Good name change here, the previous name made me think we were maybe going to get a list of string values or pojo property values or something.